Add NusaParagraph Emotion Classification

gentaiscool opened the pull request:
NusaParagraphEmotionClassification is a multi-class emotion classification on 10 Indonesian languages.

## Checklist
<!-- Please do not delete this -->

- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 


### Adding datasets checklist
<!-- see also https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md -->

**Reason for dataset addition**: ... <!-- Add reason for adding dataset here. E.g. it covers task/language/domain previously not covered -->

- [x] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb -m {model_name} -t {task_name}` command.
  - [x] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [x] `intfloat/multilingual-e5-small`
- [x] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [x] If the dataset is too big (e.g. >2048 examples), considering using `self.stratified_subsampling() under dataset_transform()`
- [x] I have filled out the metadata object in the dataset file (find documentation on it [here](https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md#2-creating-the-metadata-object)).
- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 

cc @Muennighoff @KennethEnevoldsen

KennethEnevoldsen commented on mteb/tasks/Classification/multilingual/NusaParagraphEmotionClassification.py:
```suggestion
        eval_splits=["test"],
```

KennethEnevoldsen commented on mteb/tasks/Classification/multilingual/NusaParagraphEmotionClassification.py:
size?

gentaiscool commented on mteb/tasks/Classification/multilingual/NusaParagraphEmotionClassification.py:
I updated the sizes.

gentaiscool commented:
Thank you for the review. I addressed them. I updated the avg number of character per sample, added the results, and points. I calculated the points, so there are new languages, but it only covered by other PR https://github.com/embeddings-benchmark/mteb/pull/927. So the points are 2. Plus, I added the point for the reviewer. @KennethEnevoldsen
